v2/examples/send/send_message_timeout.go

```diff
@@ -0,0 +1,27 @@
+package examples
+
+import (
+	"context"
+	"log"
+
+	"github.com/trycourier/courier-go/v2"
+)
+
+func sendMessage() {
```

Comment:
doesn't this already exist? why are we making another send method?

Reply:
It's an example. There's no impact here.